eng/Version.Details.props

```diff
@@ -45,6 +45,8 @@ This file should be imported by eng/Versions.props
     <MicrosoftDiaSymReaderPackageVersion>2.0.0</MicrosoftDiaSymReaderPackageVersion>
     <!-- dotnet/arcade-services dependencies -->
     <MicrosoftDotNetDarcLibPackageVersion>1.1.0-beta.25503.1</MicrosoftDotNetDarcLibPackageVersion>
+    <!-- dotnet/sdk dependencies -->
+    <MicrosoftDotNetFileBasedProgramsPackageVersion>10.0.200-preview.0.25556.104</MicrosoftDotNetFileBasedProgramsPackageVersion>
```

Comment:
We can probably setup a subscription so that we get pushed updates for this package automatically, but, want to start by just taking the version that was [pushed to the feed](https://dev.azure.com/dnceng/internal/_build/results?buildId=2832827&view=results) yesterday.